resources/templates/md/docs/klipse.md

```diff
@@ -0,0 +1,135 @@
+{:title "Klipse integration"
+ :layout :page
+ :page-index 8
+ :toc true
+ :section "Your Content"}
+
+[Klipse](https://github.com/viebel/klipse)
+is a tool for making code snippets on a web page interactive.
+It consists of a javascript plugin that evaluates your code snippets.
+Visitors can edit the snippets and they are evaluated continuously as they type.
+It's all evaluated client-side, so no server involved.
+
+In Cryogen you can integrate klipse in your blog with minimal effort.
+You configure the klipse plugin in `config.edn` and set `:klipse` to `true` in the post metadata.
+Alternatively you can override specific settings in the post metadata by passing a map.
+
+## Configuration
+
+The configuration is a map that may include the following keys.
+
+<table class="table table-bordered">
+<thead>
+<tr>
+<th>Key</th>
+<th>Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>`settings`</td>
+<td>Contains a map that's passed to `window.klipse_settings`.
+All the settings are documented [here](https://github.com/viebel/klipse#configuration).
+In cryogen you can use `:lisp-cased-keywords` instead of `"snake_cased_strings"`.
```

Comment:
I added the __clojure__ map wording, but I would argue that it's obvious that you can use keywords, but not that it translates from kebab-case to underscores/camelCase as necessary. It's understandable by looking at the examples, but I still like things spelled out. I put it in parentheses and added some spacing, which I feel improved it. I'll remove it if you feel like it's still confusing/unnecessary.